src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollectorHelper.java

```diff
@@ -0,0 +1,82 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.sql.Date;
+import java.util.function.Function;
+
+import org.kuali.rice.core.api.datetime.DateTimeService;
+
+import edu.cornell.kfs.concur.ConcurParameterConstants;
+
+/**
+ * Helper class providing various utility methods and properties,
+ * intended for use by the ConcurDetailLineGroupForCollector class.
+ */
+public class ConcurDetailLineGroupForCollectorHelper {
+
+    protected DateTimeService dateTimeService;
+    protected Function<String,String> dashPropertyValueGetter;
+    protected String actualFinancialBalanceTypeCode;
+    protected Date transmissionDate;
+    protected String documentTypeCode;
+    protected String systemOriginationCode;
+    protected String chartCode;
+    protected String prepaidOffsetAccountNumber;
+    protected String prepaidOffsetObjectCode;
+    protected String cashOffsetObjectCode;
+
+    public ConcurDetailLineGroupForCollectorHelper(String actualFinancialBalanceTypeCode, Date transmissionDate,
+            DateTimeService dateTimeService, Function<String,String> dashPropertyValueGetter, Function<String,String> parameterService) {

```

Comment:
should this be named something other than parameterService, so it doesn't get confused with the Rice service?